internal/osbuildexecutor/runner-impl-aws-ec2.go

```diff
@@ -24,6 +31,173 @@ func prepareSources(manifest []byte, store string, extraEnv []string, result boo
 	return err
 }
 
+func waitForSI(ctx context.Context, host string) bool {
```

Comment:
We have some similar code in https://github.com/osbuild/osbuild-composer/blob/main/cmd/osbuild-worker-executor/main_test.go#L19 - I wonde if its worthwhile to extract/reuse/unify them?

Reply:
Hmm, like something in internal/common?